pkg/controller/migcluster/validation.go

```diff
@@ -226,11 +226,23 @@ func (r *ReconcileMigCluster) validateSaTokenPrivileges(cluster *migapi.MigClust
 	}
 
 	// check for access to all verbs on all resources in all namespaces
-	// in order to determine if the service account has cluster-admin
-	sar := auth.SelfSubjectAccessReview{
+	// in the migration.openshift.io and velero.io groups in order to
+	// determine if the service account has sufficient permissions
+	migrationSar := auth.SelfSubjectAccessReview{
 		Spec: auth.SelfSubjectAccessReviewSpec{
 			ResourceAttributes: &auth.ResourceAttributes{
-				Group:    "*",
+				Group:    "migration.openshift.io",
```

Comment:
By not including other groups such as:
- "" core (pods etc)
- "apps"

Won't this defeat the purpose of the validation .. To ensure the token is a cluster admin?

Reply:
@jortel we are moving away from requiring cluster-admin and moving towards requiring the exact permissions needed by controller. This is a step towards checking the permissions on a more granular level. 

I agree with you that we need more permissions than are covered by these SAR checks.

Reply:
Understood.  Meant cluster-admin figuratively.